addons/payment_mollie_official/static/src/js/payment_mollie_official.js

```diff
@@ -0,0 +1,398 @@
+odoo.define('payment_mollie_official.mollie_payment_form', function (require) {
+"use strict";
+
+    var ajax = require('web.ajax');
+    var config = require('web.config');
+    var core = require('web.core');
+    var Dialog = require("web.Dialog");
+    var Widget = require("web.Widget");
+    var rpc = require("web.rpc");
+    var _t = core._t;
+    var PaymentForm = require('payment.payment_form');
+    
+    
+var PaymentFormMollie = PaymentForm.extend({

```

Comment:
should be an `include` and work by checking the payment form acquirer that got selected. There is a single payment form that works with multiple providers, you can't attach 2 widgets to it without causing trouble with event handling.